src/domain/forms/components/FormFieldsErrorSummary.tsx

```diff
@@ -7,6 +8,10 @@ type Props = {
 };
 
 export default function FormFieldsErrorSummary({ notificationLabel, children }: Readonly<Props>) {
+  if (Children.count(children) === 0) {
+    return null;
```

Comment:
Would returning false from component be better than returning null? https://medium.com/@davidkelley87/stop-using-return-null-in-react-a2ebf08fc9cd

Reply:
In official React docs it says to return null: https://react.dev/learn/conditional-rendering#conditionally-returning-nothing-with-null